SCP -> BTP, Cloud Foundry - Cloud Foundry environment (#199)
SAP/cloud-sdk: docs/js/getting-started.mdx

  It uses the [project "Piper" General Purpose Pipeline](https://sap.github.io/jenkins-library).
- **`pipeline_config.yml`**: Pipeline configuration file for the Jenkins pipeline.

> #### Cloud Foundry environment

Reviewer: ```suggestion
#### Cloud Foundry
```

This will start a local server in watch mode so that subsequent changes will automatically trigger a restart of the server.
Go to `http://localhost:3000` and you should get a `Hello, World!` in response.

> ## Deploy the Project on SAP BTP Cloud Foundry Environment

Reviewer: ```suggestion
## Deploy the Project on SAP BTP, Cloud Foundry Environment
```
```suggestion
## Deploy the Project to the SAP BTP Cloud Foundry Environment
```

### Prerequisites

> The Cloud Foundry environment CLI comes in handy when you want to deploy your application to the SAP Business Technology Platform.

Reviewer: ```suggestion
The Cloud Foundry CLI comes in handy when you want to deploy your application to the SAP Business Technology Platform.
```

This will start a local server in watch mode so that subsequent changes will automatically trigger a restart of the server.
Go to `http://localhost:3000` and you should get a `Hello, World!` in response.

> ## Deploy the Project on Cloud Foundry environment

Author: ```suggestion
## Deploy the Project on SAP BTP Cloud Foundry Environment
```

### Prerequisites

The Cloud Foundry environment CLI comes in handy when you want to deploy your application to the SAP Business Technology Platform.

> You can find installation instructions for all common platforms in the [`Cloud Foundry environment documentation`](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html).

Author: ```suggestion
You can find installation instructions for all common platforms in the [`Cloud Foundry documentation`](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html).
```

If you don't have an [SAP Business Technology Platform](https://account.hana.ondemand.com/) account you need to create one.
:::

> To deploy our application, we first need to login to Cloud Foundry environment in SAP Business Technology Platform using the `cf` CLI.

Reviewer: ```suggestion
To deploy our application, we first need to login to Cloud Foundry in the SAP BTP using the `cf` CLI.
```

```

This command will use your local sources for transpiling, packaging and deployment, but will omit packaging your local `node_modules` as those can be system dependent.

> Dependencies will instead be installed automatically when deploying to `Cloud Foundry environment`.

Reviewer: ```suggestion
Dependencies will instead be installed automatically when deploying to Cloud Foundry.
```

> The Cloud Foundry environment CLI will automatically pick up the `manifest.yml` of the project when deploying your application.

Reviewer: ```suggestion
The Cloud Foundry CLI will automatically pick up the `manifest.yml` of the project when deploying your application.
```

    random-route: true
```

> - The specified `path` instructs Cloud Foundry environment to upload all the files from the `deployment/` folder.

Reviewer: ```suggestion
- The specified `path` instructs Cloud Foundry to upload all the files from the `deployment/` folder.
```

cf create-service destination lite my-destination
```

> This tells `Cloud Foundry environment in SAP Business Technology Platform` to create an instance of the destination service with service plan `lite` and make it accessible under the name `my-destination`.

Reviewer: ```suggestion
This tells Cloud Foundry to create an instance of the destination service with service plan `lite` and make it accessible under the name `my-destination` on the SAP BTP.
```

}
```

> The value for `xsappname` again has to be unique across the whole of `Cloud Foundry environment in SAP Business Technology Platform`, so make sure to choose a unique name or prefix.

Reviewer: ```suggestion
The value for `xsappname` again has to be unique across the whole of SAP BTP, so make sure to choose a unique name or prefix.
```
